cpp/src/Slice/Scanner.l

```diff
@@ -61,144 +39,430 @@ namespace Slice
 {
 
 // Definitions for the case-insensitive keyword-token map.
-typedef std::map<std::string, int> StringTokenMap;
-static StringTokenMap keywordMap;
+static map<string, int> keywordMap;
```

Comment:
Our preferred style is to define non-static global variables and functions in the anonymous namespace.

Reply:
Alright so switch these from being static to being non-static in an anonymous namespace then?

Reply:
Or are you saying that I should put the variables that are non-static in an anonymous namespace and leave the static ones alone?

Reply:
Yes, if doable, move all global-namespace static variable and methods to anonymous namespace non-static variables and methods.